# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mine/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/cdb-0.75/auto-str.c.vanilla	2024-07-21 17:21:00.466805513 +0200
+++ mine-0.23/cdb-0.75/auto-str.c	2024-07-21 17:21:21.955803442 +0200
@@ -1,8 +1,9 @@
 #include <stdio.h>
+#include <unistd.h>
 
 #define puts(s) fputs(s, stdout)
 
-main(argc,argv)
+int main(argc,argv)
 int argc;
 char **argv;
 {
